examples/vg-specs/layer_precipitation_mean.vg.json

```diff
@@ -105,7 +105,7 @@
         },
         {
             "name": "layer_1_width",
-            "update": "21"
+            "update": "bandspace(domain('undefined').length, 0.1, 0.05) * 21"
```

Comment:
Note that this one actually is not perfectly correct.  
The correct behavior is that the scale should be merged (and should be handled as a part of #2198).  

In any case, the visual output looks fine for now away.